### PR TITLE
Add video ad targeting videoid and brand

### PR DESCRIPTION
**Pull request by VladDubrovskis**
/cc @lc512k @AlbertoElias

### CODE REVIEW

File: src/models/videojs.js [hunk 1]
```diff
@@ -181,10 +189,33 @@ class VideoJsPlayer extends Video {
...
 	}
 
+	getVideoBrand() {
+		if(!this.brightcoveData.tags || this.brightcoveData.tags.length === 0) {
+			return false;
+		} else {
+			let filtered = this.brightcoveData.tags.filter(val => val.toLowerCase().indexOf('brand:') !== -1);
+			if(filtered.length > 0) {
+				try {
+					 return filtered.pop().substring(6);
```
Comment (AlbertoElias) [inline src/models/videojs.js:200]: could you add a comment for the part of the string you´re looking for

File: src/models/videojs.js [hunk 2]
```diff
@@ -181,10 +189,33 @@ class VideoJsPlayer extends Video {
...
+					 return false;
+				}
+			} else {
+				return false;
+			}
+		}
+	}
+
 	advertising(player, videoIdProperty) {
+		let advertisingUrl = `http://pubads.g.doubleclick.net/gampad/ads?env=vp&gdfp_req=1&impl=s&output=xml_vast2&iu=${this.targeting.site}&sz=${this.targeting.sizes}&unviewed_position_start=1&scp=pos%3D${this.targeting.position}&ttid=${this.targeting.videoId}`;
```
Comment (AlbertoElias) [inline src/models/videojs.js:214]: It's quite hard to understander this API, could you add a comment as well?

**Comment by AlbertoElias** [comment]
Just need a couple comments to properly understand it and for future devs that might look into it, ads are hard for people outside of ads :P

**Comment by VladDubrovskis** [comment]
fair point, will add comments :)

File: src/models/videojs.js [hunk 3]
```diff
@@ -196,6 +196,7 @@ class VideoJsPlayer extends Video {
 			let filtered = this.brightcoveData.tags.filter(val => val.toLowerCase().indexOf('brand:') !== -1);
 			if(filtered.length > 0) {
 				try {
+						// when we target the value in the ad server, we only want to target actual brand name, so we strip out "brand:" part of the string
```
Comment (AlbertoElias) [inline src/models/videojs.js]: indentation

File: src/models/videojs.js [hunk 4]
```diff
@@ -208,6 +209,8 @@ class VideoJsPlayer extends Video {
 	}
 
 	advertising(player, videoIdProperty) {
+		// ad server request call that contains ad server details such as: site(iu), sizes(sz), position(pos), video id(ttid) and branding(brand) if onw is available
```
Comment (AlbertoElias) [inline src/models/videojs.js]: what´s onw?

File: src/models/videojs.js [hunk 5]
```diff
@@ -208,6 +209,8 @@ class VideoJsPlayer extends Video {
...
+		// this key values are then used on ad server to target pre roll advertising
```
Comment (AlbertoElias) [inline src/models/videojs.js]: these

**Pull request merged**